packages/dev/core/src/Misc/greasedLineTools.ts

```diff
@@ -0,0 +1,193 @@
+import { VertexBuffer } from "./../Buffers/buffer";
+import { TmpVectors, Vector3 } from "../Maths/math.vector";
+import type { AbstractMesh } from "../Meshes/abstractMesh";
+import { Curve3 } from "..";

```

Comment:
the reason the current build fails is this line - make sure to always reference the right file